darts/models/forecasting/arima.py

```diff
@@ -45,20 +53,29 @@ def __init__(
 
         Parameters
         ----------
-        p : int
-            Order (number of time lags) of the autoregressive model (AR).
+        p : int | Sequence[int]
+            Order (number of time lags) of the autoregressive model (AR). A
+            sequence of integers can be given specifying exactly which lag
+            orders are included.
         d : int
             The order of differentiation; i.e., the number of times the data
             have had past values subtracted (I).
-        q : int
-            The size of the moving average window (MA).
-        seasonal_order: Tuple[int, int, int, int]
+        q : int | Sequence[int]
+            The size of the moving average window (MA). A sequence of integers
+            can be given specifying exactly which lag orders are included.
+        seasonal_order: Tuple[int | Sequence[int], int, int | Sequence[int], int]
             The (P,D,Q,s) order of the seasonal component for the AR parameters,
-            differences, MA parameters and periodicity.
+            differences, MA parameters and periodicity. D and s are always integers,
+            while P and Q may either be integers or sequence of positive integers
+            specifying exactly which lag orders are included.
         trend: str
-            Parameter controlling the deterministic trend. 'n' indicates no trend,
-            'c' a constant term, 't' linear trend in time, and 'ct' includes both.
-            Default is 'c' for models without integration, and no trend for models with integration.
+        trend: Literal['n', 'c', 't', 'ct'] | Sequence[int], optional
+            Parameter controlling the deterministic trend. Can be specified as
+            a string where 'c' indicates a constant term, 't' indicates a
+            linear trend in time, and 'ct' includes both. Can also be specified
+            as an sequence defining a polynomial, as in `numpy.poly1d`, where
+            `[1,1,0,1]` would denote :math:`a + bt + ct^3`. Default is 'c' for
+            models without integration, and no trend for models with integration.
         add_encoders
```

Comment:
the `random_state` parameter is missing, could you add it?

Reply:
I don't see any comments in the release notes of statmodels 14.0 about this https://github.com/unit8co/darts/pull/1766. Furthermore, I searched for the diff between 13.5 and 14.0 and could not see anything related to non-reproducibility (https://github.com/statsmodels/statsmodels/compare/v0.13.5...v0.14.0). Is it parameter useful ?

Reply:
The random state is from darts to get reproducible result for probablistic forecasts.
see [here](https://www.statsmodels.org/stable/generated/statsmodels.tsa.arima.model.ARIMA.simulate.html).

I see we don't have this parameter described in the other local models either. We can do it in another PR then